pycryptobot.py

```diff
@@ -94,14 +94,15 @@ def calculateMargin(buy_size: float=0.0, buy_filled: int=0.0, buy_price: int=0.0
         print (f'sell_size: {sell_size}')
         print (f'sell_price: {sell_price}')
         print (f'sell_fee: {sell_fee}', "\n")
-
-    #buy_value = buy_size + buy_fee
-    #sell_value = sell_size - sell_fee
-    #profit = sell_value - buy_value
-
-    buy_value = buy_price + buy_fee
-    sell_value = sell_price - sell_fee
-    profit = sell_value - buy_value
+    
+    if app.getExchange() == 'coinbasepro' : 
+        buy_value = buy_size + buy_fee
+        sell_value = sell_size - sell_fee
+        profit = sell_value - buy_value
+    else :
+        buy_value = buy_price + buy_fee
```

Comment:
Are we sure this is right for Binance? buy_price + buy_fee? Surely that we should be working off the trade size and not the price. Is maybe the price coming through weirdly for Binance?

Reply:
I will merge this but I suspect a closer look of Binance is needed.

Reply:
this sure, i still can see some strange stuffs with binance, sometimes the calculated fees are ok, sometimes no